src/plugins/intel_cpu/src/nodes/executors/acl/acl_ie_scheduler.cpp

```diff
@@ -24,8 +24,7 @@ void ACLScheduler::set_num_threads(unsigned int num_threads) {}
 
 void ACLScheduler::schedule_custom(ICPPKernel *kernel, const Hints &hints, const Window &window, ITensorPack &tensors) {
     const Window & max_window = window;
-    const unsigned int num_iterations =
-            max_window.num_iterations(hints.split_dimension()) == 1 ? 1 : max_window.num_iterations_total();
+    const unsigned int num_iterations = max_window.num_iterations(hints.split_dimension());
```

Comment:
How this issue was found? Shouldn't we extend test coverage?

Reply:
this calculation of iteration is default for all schedulers in ACL, but in documents old way same work